iphone/Classes/TiUIWindowProxy.m

```diff
@@ -304,6 +304,10 @@ - (void)viewWillAppear:(BOOL)animated;    // Called when the view is about to ma
 - (void)viewDidAppear:(BOOL)animated;     // Called when the view has been fully transitioned onto the screen. Default does nothing
 {
 	[self updateTitleView];
+    [self updateHidesBarsOnSwipe];
+    [self updateHidesBarsOnTap];
+    [self updateHidesBarsWhenVerticallyCompact];
+    [self updateHidesBarsWhenKeyboardAppears];
```

Comment:
Indentation again. And do we need to call these on appear as well? I would expect that is already gets set by the setters used and the default behavior for the remaining ones.